### PR TITLE
fix makefile dependencies

### PR DESCRIPTION
**Pull request by IdanTo1**
make a complete makefile for the project

Signed-off-by: Idan Tomer <idan.tomer2@gmail.com>

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -8,31 +8,31 @@ LFLAGS  = -L/usr/lib
...
+bin/PopularTag.o: include/PopularTag.h src/PopularTag.cpp
 	$(CC) $(CFLAGS) -c -Iinclude -o bin/PopularTag.o src/PopularTag.cpp
 
 
 
 #Clean the build directory
-.PHONY: clean
+.PHONY: all clean
 clean:
 	rm -f bin/*
```
Comment (Hroy02) [inline makefile]: ?

**Review by Hroy02 — CHANGES_REQUESTED** [review]
Looks great!

File: makefile (hunk 1 above)
Comment (IdanTo1) [reply makefile]: you don't have a file named all created, it's just a code to run all makes
and so it is phony

**Review by Hroy02 — DISMISSED** [review]

**Review by Hroy02 — APPROVED** [review]
Perfect

**Pull request merged**